gatsby-source-graphcms/README.md

```diff
@@ -140,9 +140,27 @@ To query for nodes from a specific Content Stage, use the `filter` query argumen
 }
 ```
 
-### Downloading local image assets
+### Usage with `gatsby-plugin-image`
 
-This source plugin provides the option to download and cache GraphCMS assets in your Gatsby project. This enables you to use [`gatsby-plugin-image`](https://www.gatsbyjs.com/plugins/gatsby-plugin-image), for image loading optimizations, with your GraphCMS image assets.
+This source plugin supports `gatsby-plugin-image` for responsive, high performance GraphCMS images direct from our CDN.
+
+Use the `gatsbyImageData` resolver on your `GraphCMS_Asset` nodes.
+
+```gql
+{
+  allGraphCmsAsset {
+    nodes {
+      gatsbyImageData(layout: FULL_WIDTH)
+    }
+  }
+}
+```
+
+For more information on using `gatsby-plugin-image`, please see the [documentation](https://www.gatsbyjs.com/plugins/gatsby-plugin-image/).
```

Comment:
It would be good if you could list the arguments that the resolver accepts, as it's not the same as the ones in the sharp plugin.

Reply:
@ascorbic Can we just link to them [here](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#image-options)?

Reply:
No, because they'll be different. Those include all of the sharp options, but your resolver accepts fewer of those. You're welcome to copy the descriptions of the ones that you do support.

Reply:
@ascorbic Updated.